src/app/utils/ProxifyUrl.js

```diff
@@ -0,0 +1,24 @@
+/*global $STM_Config:false*/
+
+//this regular expression should capture all possible proxy domains
+const rProxyDomains = /http(s)?:\/\/steemit(dev|stage)?images.com\/([0-9]+x[0-9]+)?\//g
+
+/**
+ * Strips all but final 'steemit' image proxy from the beginning of the url.
```

Comment:
i think it should strip all of them, then add just the one that is specified in the condenser config

Reply:
This comment is (now) incorrect, what you describe is what happens.